_std/defines/projectiles.dm

```diff
@@ -32,9 +32,11 @@
 
 
 //ammo categories
+#define AMMO_SHOTGUN_SCRAP "shotgun_scrap"
```

Comment:
Personal take, I would like to see it usable in all shotguns. Non-antags can obtain riot shotguns out in the debris field & azones, why force them to only use the slam gun / coach gun if they're able to obtain something better?

Not an issue for antags considering they can already break into the armory or purchase better ammo from uplink / syndicate listening post.